index.js

```diff
@@ -0,0 +1,49 @@
+const $SLIDER = document.querySelector('.location__wrapper');
+
+const prevSlideBtn = $SLIDER.querySelector('.slide-prev');
+const nextSlideBtn = $SLIDER.querySelector('.slide-next');
+const slides = $SLIDER.querySelectorAll('.slider__item');
+
+const sliderDots = document.querySelectorAll('.slider__dot');
+let currentSlideIndex = 0;
+
+const showPrevSlide = function () {
+  hideSlide(currentSlideIndex);
+  if (currentSlideIndex == 0) {
+    currentSlideIndex = slides.length - 1;
+    showSlide(currentSlideIndex);
+  } else {
+    currentSlideIndex--;
+    showSlide(currentSlideIndex);
+  }
+};
+const nextPrevSlide = function () {
```

Comment:
every function should be called with verb for example viewPreviousSlide. please fix naming, replace with arrow function and remove all magic numbers here